### ISSUE TITLE
[Performance Issue]: h.5 isn't supported

### ISSUE THREAD

**Issue by Naonao9527** [SciSharp/TensorFlow.NET#1266]
### Brief Description

Unhandled exception. System.NotImplementedException: Model load of h5 format has not been supported. Please submit an issue to https://github.com/SciSharp/TensorFlow.NET/issues if it's needed.

### Device and Context

.net8 
tensorflow 2.150
win-X64

**Comment by Hoai-Phong** [comment]
Do you have a fix? @Naonao9527

**Comment by Naonao9527** [comment]
not yet,I use tensorflow of python instesd.

**Comment by Hoai-Phong** [comment]
@Naonao9527  can you share it with me?
I see net 6 only. net not ok

**Comment by Naonao9527** [comment]
The .h5 file was created by python,I wanted to load it in .net,Maybe it's the reason why it don't work.you can try it again in your computer.

**Comment by Hoai-Phong** [comment]
yes, my h5 file is also read from python, but when read through net 8, it has the same error. 
I can't read h5 file from net
